// Arguments for results of callback function of the network layer
package net;

message NetStartResult {
  required bytes result = 1;
  optional bytes ip = 2;
  optional int32 port = 3;
};
